src/security-rules/security-rules.ts

```diff
@@ -38,21 +38,6 @@ export interface RulesetMetadata {
   readonly createTime: string;
 }
 
-interface Release {
-  readonly name: string;
-  readonly rulesetName: string;
-  readonly createTime: string;
-  readonly updateTime: string;
-}
-
-interface RulesetResponse {
-  readonly name: string;
-  readonly createTime: string;
-  readonly source: {
-    readonly files: RulesFile[];
-  };
-}
-
```

Comment:
Are you removing these in favor of defining them in `security-rules/security-rules-api-client.ts` because they are set by the API?

Reply:
Correct. They are being moved to the api-client module, which in this case is treated as an internal implementation module.